### PR TITLE
Readds coverage, doc deploy, MRO database fix

### PR DESCRIPTION
**Pull request by jlaura**
Not sure when these got removed, but this PR readds deploy of coverage reports to coveralls and deploy of docs to http://usgs-astrogeology.github.io/plio/

For the doc redeploy, if after a merge the CSS is still not loading, I will do another PR.

### CODE REVIEW

**Comment by jlaura** [comment]
@all please squash and merge if this looks good.

**Comment by jlaura** [comment]
p.s. I need to rebase...

File: plio/io/hcube.py [hunk 1]
```diff
@@ -2,8 +2,11 @@
 import gdal
 
 from ..utils.indexing import _LocIndexer, _iLocIndexer
+<<<<<<< HEAD
```
Comment (tthatcher95) [inline plio/io/hcube.py]: @jlaura Git markers were left after commits

File: plio/utils/indexing.py [hunk 2]
```diff
@@ -35,27 +35,54 @@ def expanded_indexer(key, ndim):
 class _LocIndexer(object):
     def __init__(self, data_array):
         self.data_array = data_array
+<<<<<<< HEAD
```
Comment (tthatcher95) [inline plio/utils/indexing.py]: @jlaura Git markers were left after commits, needs to be removed

**Pull request closed without merging**